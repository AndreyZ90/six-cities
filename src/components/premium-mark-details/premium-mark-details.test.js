import React from 'react';
import TestRenderer from 'react-test-renderer';

import PremiumMarkDetails from '@/components/premium-mark-details/premium-mark-details';

describe(`PremiumMarkDetails component snapshot`, () => {
  test(`Should correctly render PremiumMarkDetails component`, () => {
    const tree = TestRenderer.create(<PremiumMarkDetails />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
