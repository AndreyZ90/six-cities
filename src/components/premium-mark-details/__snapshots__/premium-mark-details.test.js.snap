// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PremiumMarkDetails component snapshot Should correctly render PremiumMarkDetails component 1`] = `
<div
  className="property__mark"
>
  <span>
    Premium
  </span>
</div>
`;
